src/service/cli/server.js

```diff
@@ -11,47 +10,22 @@ module.exports = {
     const [enteredPort] = args;
     const port = Number.parseInt(enteredPort, 10) || DEFAULT_PORT;
 
-    const sendResponse = (res, statusCode, message) => {
-      const template = `
-        <!Doctype html>
-          <html lang="ru">
-          <head>
-            <title>With love from Node</title>
-          </head>
-          <body>${message}</body>
-        </html>`.trim();
+    const app = express();
+    app.use(express.json());
 
-      res.writeHead(statusCode, {
-        'Content-Type': `text/html; charset=UTF-8`,
-      });
+    app.get(`/service`, async (req, res) => {
```

Comment:
```suggestion
    app.get(`/posts`, async (req, res) => {
```
В задании нужно так. Но так как изменений не много, подправишь в некст ПР.

Reply:
Точно, хорошо, поправлю